qiskit/compiler/assembler.py

```diff
@@ -154,3 +161,156 @@ def assemble_circuits(circuits, run_config=None, qobj_header=None, qobj_id=None)
+                       qobj_header=None, qobj_id=None, schedule_los=None,
+                       shots=1024, max_credits=10, seed=None,
+                       meas_level=2, meas_return='avg', memory_slots=None,
+                       memory_slot_size=100, rep_time=None,
+                       instruction_converter=PulseQobjConverter,
+                       **run_config):
+    """Assembles a list of circuits into a qobj which can be run on the backend.
+
+    Args:
+        schedules (list[Schedule] or Schedule): schedules to assemble
+        default_qubit_los (list): List of default qubit lo frequencies
+        default_meas_los (list): List of default meas lo frequencies
+        qobj_header (QobjHeader): header to pass to the results
+        qobj_id (int): identifier for the generated qobj
+        schedule_los(None or list[Union[Dict[OutputChannel, float], LoConfig]] or
+                        Union[Dict[OutputChannel, float], LoConfig]): Experiment LO configurations
+        shots (int): number of repetitions of each circuit, for sampling
+        max_credits (int): maximum credits to use
+        seed (int): random seed for simulators
+        meas_level (int): set the appropriate level of the measurement output.
+        meas_return (str): indicates the level of measurement information to return.
+            "single" returns information from every shot of the experiment.
+            "avg" returns the average measurement output (averaged over the number of shots).
+            If the meas level is 2 then this is fixed to single.
+        memory_slots (int): number of classical memory slots used in this job.
+        memory_slot_size (int): size of each memory slot if the output is Level 0.
+        rep_time (int): repetition time of the experiment in μs.
+            The delay between experiments will be rep_time.
+            Must be from the list provided by the device.
+        instruction_converter (PulseQobjConverter): converter for pulse instruction
+        run_config: Additional keyword arguments to be inserted in the Qobj configuration.
+
+    Returns:
+        PulseQobj: the Qobj to be run on the backends
+
+    Raises:
+        QiskitError: when invalid schedules or configs are provided
+    """
+    if isinstance(schedules, Schedule):
+        schedules = [schedules]
+
+    # add default empty lo config
+    if schedule_los is None:
+        schedule_los = []
+
+    if isinstance(schedule_los, (LoConfig, dict)):
+        schedule_los = [schedule_los]
+
+    if qobj_id is None:
+        qobj_id = str(uuid.uuid4())
+
+    # Convert to LoConfig if lo configuration supplied as dictionary
+    schedule_los = [lo_config if isinstance(lo_config, LoConfig) else LoConfig(lo_config)
+                    for lo_config in schedule_los]
+
+    user_pulselib = set()
+
+    qobj_header = qobj_header or QobjHeader()
+    # create run configuration and populate
+    run_config = {}
```

Comment:
the `assemble_schedules` function accepts a `**run_config`, but i think it gets overwritten here and never used.